scripts/Utility.js

```diff
@@ -0,0 +1,177 @@
+const Util = require('fallout-utility');
+const Path = require('path');
+const { Config, Language } = require('./config');
+const { SafeMessage, SafeInteract } = require('./safeActions');
+const LoadScripts = require('./loadScripts');
+const CommandPermission = require('./commandPermissions');
+const MemberPermissions = require('./memberPermissions');
+const registerInteractionCommmands = require('./registerInteractionCommands');
+
+module.exports = class AxisUtility {
+    /**
+     * 
+     * @param {Object[]} options - options
+     * @param {string} [options.logger=new Util.Logger()] - Logger object
+     * @param {Object[]} options.config - Bot config.yml
+     * @param {Object[]} options.language - Language config.yml
+     * @param {Object[]} options.Client - Client object
+     */
+    constructor(options = { logger: new Util.Logger('AxisUtility'), config: null, language: null, Client: null }) {
+        if(!options.config) throw new TypeError('No config provided');
+        if(!options.language) throw new TypeError('No language provided');
+        if(!options.Client) throw new TypeError('No client provided');
+
+        this.logger = options.logger;
+        this.config = options.config;
+        this.language = options.language;
+        this.scripts = {};
+        this.commands = { MessageCommands: [], InteractionCommands: [] };
+        this.Client = options.Client;
+    }
```

Comment:
Allow setting the commands and scripts optionally
```suggestion
    constructor(options = { logger: new Util.Logger('AxisUtility'), config: null, language: null, Client: null, scripts: [], commands: { MessageCommands: [], InteractionCommands: [] } }) {
        if(!options.config) throw new TypeError('No config provided');
        if(!options.language) throw new TypeError('No language provided');
        if(!options.Client) throw new TypeError('No client provided');

        this.logger = options.logger;
        this.config = options.config;
        this.language = options.language;
        this.scripts = options.scripts?.length ? options.scripts : [];
        this.commands = options.commands ? options.commands : { MessageCommands: [], InteractionCommands: [] };
        this.Client = options.Client;
    }
```